src/test/java/com/powsybl/openloadflow/ac/AreaInterchangeControlTest.java

```diff
@@ -107,19 +150,38 @@ void remainingMismatchDistributeOnReferenceGenerator() {
 
         // falls back to FAIL
         assertEquals(Double.NaN, network.getGenerator("g1").getTerminal().getP(), 1e-3);
-        assertEquals(0, mainComponentResult.getDistributedActivePower(), 1e-3);
         assertEquals(-30, mainComponentResult.getSlackBusResults().get(0).getActivePowerMismatch(), 1e-3);
+        return mainComponentResult;
     }
 
     @Test
-    void remainingMismatchThrow() {
+    void acRemainingMismatchThrow() {
+        remainingMismatchThrow();
+    }
+
+    @Test
+    void dcRemainingMismatchThrow() {
+        parameters.setDc(true);
+        remainingMismatchThrow();
+    }
+
+    private void remainingMismatchThrow() {
         parametersExt.setSlackDistributionFailureBehavior(OpenLoadFlowParameters.SlackDistributionFailureBehavior.THROW);
         Network network = MultiAreaNetworkFactory.createOneAreaBase();
         network.getGenerator("g1").setMinP(90); // the generator should go down to 70MW to meet the interchange target
         CompletionException thrown = assertThrows(CompletionException.class, () -> loadFlowRunner.run(network, parameters));
         assertEquals("Failed to distribute interchange active power mismatch", thrown.getCause().getMessage());
     }
 
+    static Stream<Arguments> slackDistributionFailureBehaviors() {
```

Comment:
could be simplified to
return Arrays.stream(OpenLoadFlowParameters.SlackDistributionFailureBehavior.values()).map(Arguments::of);

but seems unused - to be deleted ?

Reply:
Yes it was unused, removed in [59fd886](https://github.com/powsybl/powsybl-open-loadflow/pull/1099/commits/59fd8860e4c54a93d7506096321ac2b0f04d5d2e)